sylvia-derive/src/message.rs

```diff
@@ -901,15 +901,28 @@ impl<'a> GlueMessage<'a> {
             let ContractMessageAttr {
                 variant,
                 has_custom_msg,
+                has_custom_query,
```

Comment:
I am wondering if we could merge those `has_custom_*` in single structure like:

```rust
struct Customs {
  has_msg: bool,
  has_query: bool,
}
```

Two fields are not much, but I am wondering if it would grow in the future. For now, please don't change it, but things to keep in mind when this grows too much.